transform/snowflake-dbt/models/incident_collaboration/incident_daily_details.sql

```diff
@@ -1,9 +1,9 @@
 {{config({
     "materialized": "incremental",
     "schema": "incident_collaboration",
-    "tags":"hourly",
+    "tags":"nightly",
     "unique_key":"id",
-    "warehouse":"ANALYST_XS"
+    "warehouse":"TRANSFORM_XS"
```

Comment:
Let's revert this change (and similar ones) and move it to the PR addressing [MM-52376](https://mattermost.atlassian.net/browse/MM-52376)